packages/admin/admin/src/form/fields/CheckboxField.tsx

```diff
@@ -1,22 +1,34 @@
-import { FormControlLabel, FormControlLabelProps } from "@mui/material";
+import { Checkbox, CheckboxProps, FormControlLabel, FormControlLabelProps } from "@mui/material";
 
-import { FinalFormCheckbox, FinalFormCheckboxProps } from "../Checkbox";
 import { Field, FieldProps } from "../Field";
 
 export interface CheckboxFieldProps extends FieldProps<string, HTMLInputElement> {
     fieldLabel?: string;
     componentsProps?: {
         formControlLabel?: FormControlLabelProps;
-        finalFormCheckbox?: FinalFormCheckboxProps;
+        formCheckbox?: CheckboxProps;
```

Comment:
Yes this is a breaking change.

@fichtnerma why are changes to this file necessary at all? 
Is the `CheckboxField` causing problems in the usage somewhere? (if so, where exactly?)

Reply:
I assume it's also partially due to with the initial values not showing. 
But I'm not sure why the other changes are needed.

Reply:
I changed the type of the props from FinalFormCheckboxProps to CheckboxProps because the `FinalFormCheckbox` isn't used anymore in this file. But FinalFormCheckboxProps is basically just a wrapper for the CheckboxProps anyway.